content_scripts/vimium_frontend.coffee

```diff
@@ -5,7 +5,7 @@
 # "domReady".
 #
 
-findModeQuery = { rawQuery: "", matchCount: 0 }
+window.findModeQuery = { rawQuery: "", matchCount: 0 }
 findModeQueryHasResults = false
 findModeAnchorNode = null
 findModeInitialRange = null
```

Comment:
Agreed, I was tempted to do that in this PR, but I held off for the sake of the diff. This is one of the things I want to do when I refactor `FindMode` later.